pkg/cmd/server/api/helpers.go

```diff
@@ -73,9 +80,13 @@ func (fl *FeatureList) Delete(items ...string) {
 // Has returns true if given feature exists in feature list. The check is
 // case-insensitive.
 func (fl FeatureList) Has(feature string) bool {
-	lowerCased := strings.ToLower(feature)
+	normalized, featureKnown := NormalizeOpenShiftFeature(feature)
```

Comment:
if `NormalizeOpenShiftFeature` returned the lowercased name even for unknown things, this would be simpler:
```
featureNormalized, _ := NormalizeOpenShiftFeature(feature)
for _, item := range fl {
  itemNormalized, _ := NormalizeOpenShiftFeature(item)
  if itemNormalized == featureNormalized {
    return true
  }
}
```